mysociety/local-intelligence-hub
# WIP build out a flat version of UI

## Description
This PR is my work so far on building out the Meep UI based on the product design here
https://www.figma.com/file/o2g2uM7bGMTK5YahLAvTV4/Mapped-Product-Design-(shadcn)?type=design&node-id=707-35271&mode=design&t=ni4o7N1TmLRQf2nT-0

## Motivation and Context
Addresses issue [MEEP-186](https://linear.app/commonknowledge/issue/MEEP-186/build-out-a-flat-version-of-the-ui-from-arbys-design)

## Testing
- Download the branch and run locally.
- Observe the home page and layout on other pages linked from the home page
- Navigate to http://localhost:3000/reports and observe the reports page
- If possible create an account on the backend, log in and observe the different nav menus when logged in vs out 

## Screenshots (if appropriate):
Home page logged out
![Untitled](https://github.com/mysociety/local-intelligence-hub/assets/43313455/3908936f-4e82-4d0b-b57a-fe3d2f238eb6)

Reports page logged in
![Untitled 2](https://github.com/mysociety/local-intelligence-hub/assets/43313455/062dcf8b-5029-4af6-8aee-7e04490c0ced)


## Types of changes
